pkg/odo/cli/component/common_link.go

```diff
@@ -114,7 +115,22 @@ func (o *commonLinkOptions) run() (err error) {
 		return err
 	}
 
-	log.Successf("%s %s has been successfully %sed from the component %s", linkType, o.suppliedName, o.operationName, o.Component())
+	log.Successf("%s %s has been successfully %sed from the component %s\n", linkType, o.suppliedName, o.operationName, o.Component())
+
+	secret, err := o.Client.GetSecret(o.secretName, o.Project)
+	if err != nil {
+		return err
+	}
```

Comment:
I think there could be a better way to handle an error to get a secret instead of just returning it after the actual linking operation has succeeded but fetching `secret` has failed. I think it's a corner case. @kadel @girishramnani @mik-dass any suggestions?

Reply:
I think returning the error is the simplest way to inform the user or maybe use a customized error message.

Reply:
@dharmit if we cannot list the secrets then yeah I think a warning can be printed. But also if we cannot list the secrets then wouldn't the cluster be in a state where linking would also fail? ( permission error or system failure )

Reply:
@mik-dass so you mean that what I've done above is OK?

@girishramnani exactly. And that seems like an edge case to me. Also, the kind of edge case that we can do nothing about but simply tell the user that something failed. Because in a step before that, we get env vars from the OpenShift cluster but in this, we would fail to get a secret. Thus, I couldn't think of a better way than what's there. :smile:

Reply:
I think it's OK, maybe just let the user know that the actual linking succeeded right after that operation is over.

Reply:
That's already happening because I have not changed anything in https://github.com/openshift/odo/pull/2035/files#diff-b69279252c95e3ee7369b8d06af6a0ebR118

Reply:
Cool :+1: